UnitsNet.Tests/CustomCode/ElectricCurrentGradientTests.cs

```diff
@@ -29,5 +29,8 @@ public class ElectricCurrentGradientTests : ElectricCurrentGradientTestsBase
     {
         protected override bool SupportsSIUnitSystem => false;
         protected override double AmperesPerSecondInOneAmperePerSecond => 1;
+        protected override double AmperePerMillisecondInOneAmperePerSecond => 1e03;
```

Comment:
I think you got the conversion functions inverted. This should be 1e-3, right?

https://www.google.com/search?q=AmperePerMillisecondInOneAmperePerSecond&oq=AmperePerMillisecondInOneAmperePerSecond&aqs=chrome..69i57.270j0j7&sourceid=chrome&ie=UTF-8

Reply:
Same for the other two.

Reply:
I thought the same thing, but when I had them as 1e-3, 1e-6, and 1e-9 the UnitTests would fail.

```  UnitsNet.Tests -> C:\Users\Anthony\source\repos\UnitsNet\Artifacts\UnitsNet.Tests\netcoreapp2.1\UnitsNet.Tests.dll
[xUnit.net 00:00:06.31]     UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.ToUnit [FAIL]
  Failed UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.ToUnit [5 ms]
  Error Message:
   Values are not equal within relative tolerance: 0.0010%
Expected: 1E-06
Actual: 1000000
Diff: 99,999,999,999,900.0000%
Expected: True
Actual:   False
  Stack Trace:
     at UnitsNet.Tests.AssertEx.EqualTolerance(Double expected, Double actual, Double tolerance, ComparisonType comparisonType) in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\AssertEx.cs:line 20
   at UnitsNet.Tests.ElectricCurrentGradientTestsBase.ToUnit() in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\GeneratedCode\TestsBase\ElectricCurrentGradientTestsBase.g.cs:line 197
[xUnit.net 00:00:06.34]     UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.AmperePerSecondToElectricCurrentGradientUnits [FAIL]
  Failed UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.AmperePerSecondToElectricCurrentGradientUnits [1 ms]
  Error Message:
   Values are not equal within relative tolerance: 0.0010%
Expected: 1E-06
Actual: 1000000
Diff: 99,999,999,999,900.0000%
Expected: True
Actual:   False
  Stack Trace:
     at UnitsNet.Tests.AssertEx.EqualTolerance(Double expected, Double actual, Double tolerance, ComparisonType comparisonType) in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\AssertEx.cs:line 20
   at UnitsNet.Tests.ElectricCurrentGradientTestsBase.AmperePerSecondToElectricCurrentGradientUnits() in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\GeneratedCode\TestsBase\ElectricCurrentGradientTestsBase.g.cs:line 124
[xUnit.net 00:00:06.38]     UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.As [FAIL]
  Failed UnitsNet.Tests.CustomCode.ElectricCurrentGradientTests.As [1 ms]
  Error Message:
   Values are not equal within relative tolerance: 0.0010%
Expected: 1E-06
Actual: 1000000
Diff: 99,999,999,999,900.0000%
Expected: True
Actual:   False
  Stack Trace:
     at UnitsNet.Tests.AssertEx.EqualTolerance(Double expected, Double actual, Double tolerance, ComparisonType comparisonType) in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\AssertEx.cs:line 20
   at UnitsNet.Tests.ElectricCurrentGradientTestsBase.As() in C:\Users\Anthony\source\repos\UnitsNet\UnitsNet.Tests\GeneratedCode\TestsBase\ElectricCurrentGradientTestsBase.g.cs:line 168
[xUnit.net 00:00:15.49]     UnitsNet.Tests.CustomCode.MassConcentrationTests.DefaultSIUnitIsKgPerCubicMeter [SKIP]
  Skipped UnitsNet.Tests.CustomCode.MassConcentrationTests.DefaultSIUnitIsKgPerCubicMeter [1 ms]

Failed!  - Failed:     3, Passed:  7400, Skipped:     1, Total:  7404, Duration: 19 s - UnitsNet.Tests.dll (netcoreapp2.1)
```

Also the code seems to be correct based off some output from my application.

  | Name | Value | Type
-- | -- | -- | --
◢ | edgerate | {100 A/μs} | UnitsNet.ElectricCurrentGradient
  | AmperePerMicrosecond | 100 | double
  | AmperePerMillisecond | 0.1 | double
  | AmperePerNanosecond | 100000 | double
  | AmperesPerSecond | 0.0001 | double
  | ▶ Dimensions | {[Time][Current]} | UnitsNet.BaseDimensions
  | ▶ QuantityInfo | {UnitsNet.QuantityInfo<UnitsNet.Units.ElectricCurrentGradientUnit>} | UnitsNet.QuantityInfo<UnitsNet.Units.ElectricCurrentGradientUnit>
  | Type | ElectricCurrentGradient | UnitsNet.QuantityType
  | Unit | AmperePerMicrosecond | UnitsNet.Units.ElectricCurrentGradientUnit
  | ▶ UnitsNet.IQuantity.QuantityInfo | {UnitsNet.QuantityInfo<UnitsNet.Units.ElectricCurrentGradientUnit>} | UnitsNet.QuantityInfo {UnitsNet.QuantityInfo<UnitsNet.Units.ElectricCurrentGradientUnit>}
  | UnitsNet.IQuantity.Unit | AmperePerMicrosecond | System.Enum {UnitsNet.Units.ElectricCurrentGradientUnit}
  | Value | 100 | double
  | _unit | AmperePerMicrosecond | UnitsNet.Units.ElectricCurrentGradientUnit?
  | _value | 100 | double